frontend/app/view/term/termwrap.ts

```
@@ -463,15 +478,104 @@ export class TermWrap {
+        if (this.pendingCompositionSpace == null) {
+            return;
+        }
+        clearTimeout(this.pendingCompositionSpace.timeout);
+        this.pendingCompositionSpace = null;
+    }
+
+    isLikelyCompositionText(data: string): boolean {
+        if (data.length === 0) {
+            return false;
+        }
+        if (/[\x00-\x1F\x7F]/.test(data)) {
```

ESLint

[error] Unexpected control character(s) in regular expression: \x00, \x1f.

(no-control-regex)